src/components/core/Button/Button.tsx

```diff
@@ -18,7 +18,7 @@ const alphaHex = (percentage: number) => {
 };
 
 type StyledButtonProps = {
-  variant?: 'primary' | 'secondary' | 'error';
+  variant?: 'primary' | 'secondary' | 'warning';
```

Comment:
renamed the `error` variant to `warning` to reflect the usage more accurately. this variant was only being used in the wallet settings section.